docs/debug-functions.md

```diff
@@ -35,10 +35,17 @@ INFO[0000] Check the deployment status executing 'kubeless function ls foo'
 $ kubeless function ls
 NAME 	NAMESPACE	HANDLER  	RUNTIME  	DEPENDENCIES	STATUS
 foo  	default  	hello,foo	python3.6	            	MISSING: Check controller logs
-$ kubectl logs -n kubeless -l kubeless=controller
-time="2018-04-27T15:12:28Z" level=info msg="Processing update to function object foo Namespace: default" controller=cronjob-trigger-controller
-time="2018-04-27T15:12:28Z" level=error msg="Function can not be created/updated: failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
-time="2018-04-27T15:12:28Z" level=error msg="Error processing default/foo (will retry): failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+$ kubectl logs -n kubeless -l kubeless=controller -c kubeless-function-controller
+time="2020-10-01T01:48:29Z" level=info msg="Processing change to Function default/foo" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Function can not be created/updated: failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Error processing default/foo (will retry): failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=info msg="Processing change to Function default/foo" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Function can not be created/updated: failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Error processing default/foo (will retry): failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=info msg="Processing change to Function default/foo" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Function can not be created/updated: failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Error processing default/foo (giving up): failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+ERROR: logging before flag.Parse: E1001 01:48:29.164740       1 function_controller.go:185] failed: incorrect handler format. It should be module_name.handler_name
```

Comment:
Can you remove these lines (41-48)? so it's easier to spot the issue

Reply:
ok,done.